docs/apis/mos/raw_data_pull_api_tokenv2_overview.md

```diff
@@ -0,0 +1,91 @@
+| `from_install_time` | string  | Start date for filtering by install time.                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | `"2022-04-22"`, `"2022-04-22 21:45"`, `"2022-04-22 21:45:23"`                                                                                            |
+| `to_install_time`   | string  | End date for filtering by install time.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | `"2022-04-22"`, `"2022-04-22 21:45"`, `"2022-04-22 21:45:23"`                                                                                            |
+| `agency`            | string  | Filter by specific agency or multiple agencies. For non-transparent agencies, using both the `agency` and `media_source` filters together may return data from all agencies, regardless of the filtered media sources.                                                                                                                                                                                                                                                                                                 | `"agency=transparent_agency"`, `"media_source=media-source-name&agency=transparent_agency"`, `"agency=transparent_agency&agency=non_transparent_agency"` |
+| `additional_fields` | array   | For the list of the additional fields and their description see the [Raw data field dictionary](https://support.appsflyer.com/hc/en-us/articles/208387843-Raw-data-field-dictionary#rawdata-fields-dictionary) table.                                                                                                                                                                                                                                                                                                  | `["blocked_reason_rule","store_reinstall","impressions"]`                                                                                                |
```

Comment:
I think we need to be more specific here. The table in the KB is very hard to understand.
We need to list it for every endpoint